test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,25 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() {
+        //empty list
+        assertNotNull();
+        //list with null
+        assertAnyNull(1,null);
+        //list with no null
+        assertNotNull(1,2,3);
+    }
+
     private void assertAreUnique(Object... objects) {
         assertTrue(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
 
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertAnyNull(Object... objects) { assertTrue(Utils.isAnyNull(objects));}
+
+    private void assertNotNull(Object... objects) { assertFalse(Utils.isAnyNull(objects));}
```

Comment:
:frowning_face: This is against our [Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) (which requires the code to follow _egyptian style_ braces). There should be a line break after `{` and before `}`